### DISCUSSION TITLE
SAFE procedure

### DISCUSSION THREAD

**Discussion by Yuanyuanhu1990** [asreview/asreview#1774] [Q&A]
I have some questions about using the SAFE procedure (https://link.springer.com/content/pdf/10.1186/s13643-024-02502-7.pdf)

1. Phase 1 Screen a random set for training data: Do you have step-by-step procedures for this phase? I am confused, for example:
Where can this random set be screened? In Zotero? or Excel? I guess this is not done in ASReview LAB, which need 5 relevant records to set up prior knowledge?
How many records should be screened (e.g., 1% of the total dataset)?
What is the purpose of this phase?
What do you mean by training data? If I've already have 5 relevant records, can I skip this phase?
2. Phase 4 Evaluate quality. The 10 highest- and lowest-ranked records from the previous phase were used to train the model; The second screeener screened the irrelevant records to identify any relevant records with the stopping rule that no extra relevant records were identified in the last 50 records; The second screeener screened all relevant records to identify any irrelevant records. 
But to calculate the interrater reliability, two screeners trained the same model and screened independently in ASReview LAB. When calculate Kappa, I guess the two screeners will go through all the disagreed records. In this case,  are there overlaps between this and phase 4? Do we need both?

**Comment by Rensvandeschoot** [comment]
Thank you for your questions! 

1) 
You can create a random set via the `sample`  function of [ASReview-Datatools](https://github.com/asreview/asreview-datatools?tab=readme-ov-file#sample). 

It's up to you which software you screen this set in. In ASReview, you can also [randomly](https://github.com/asreview/asreview-datatools?tab=readme-ov-file#sample) screen papers, but you need at least one relevant and one irrelevant record to start. In version 2, you can begin screening without any prior knowledge!

I suggest combining the training and calibration phase with the first phase of the SAFE procedure.
With training, we refer to the training of the screeners. 

2) 
You can use active learning in the screening phase in many ways. One option is asking two screeners to use the same data and model. Then, you can calculate the inter-rater reliability, for example, by using the [ASReview-IRR package](https://github.com/langtonhugh/asreview_irr) by @langtonhugh. Other ways to collaborate are explained in a separate thread: https://github.com/asreview/asreview/discussions/975. 

Checking the excluded records can be helpful if you want to [double-check exclusions](https://www.crd.york.ac.uk/prospero/display_record.php?RecordID=494027), if you are processing an [enormous dataset](https://osf.io/preprints/psyarxiv/j6nqz), or if you need to correct [noisy labels](https://systematicreviewsjournal.biomedcentral.com/articles/10.1186/s13643-024-02472-w). 

And, yes, you should always check the disagreements! 

Hopefully, my answers are helpful, but please let me know if I can further help!
